src/OSPSuite.Infrastructure.Import/Core/DataSource.cs

```diff
@@ -156,6 +156,9 @@ public bool ValidateDataSource(IReadOnlyList<ColumnInfo> columnInfos, IDimension
                      if (errorColumn.Value != null && measurementColumn.Value.Count != errorColumn.Value.Count)
                         throw new OSPSuiteException(Error.MismatchingArrayLengths);
```

Comment:
how is it possible that `measurementColumn.Value.Count != errorColumn.Value.Count` ?
Or is it when columns are created "manually" e.g. from R?